pkg/cmd/pipelinerun/list.go

```diff
@@ -108,6 +109,9 @@ List all PipelineRuns in a namespace 'foo':
 			}
 
 			if prs != nil {
+				if opts.Reverse {
```

Comment:
Checking `opts.Reverse` here misses out on situations when `--output` or `-o` would be used. 

You may want to check for `if opts.Reverse` on [line 88](https://github.com/tektoncd/cli/blob/64f3fa7c03b43e3d75d8de86adacd9728b794259/pkg/cmd/pipelinerun/list.go#L87). This way `--output` situations would also be included. You would also need a nil check for `prs`:

```go
if prs != nil && opts.Reverse {
    reverse(prs)
}
```